src/printing.jl

```diff
@@ -79,18 +79,52 @@ function Base.show(io::IO, results::OperationsProblemResults)
 
 function Base.show(io::IO, ::MIME"text/plain", results::IS.Results)
     println(io, "\nResults")
-    println(io, "===============\n")
-
-    for (k, v) in results.variable_values
-        time = DataFrames.DataFrame(Time = results.time_stamp[!, :Range])
-        if size(time, 1) == size(v, 1)
-            var = hcat(time, v)
-        else
-            var = v
+    println(io, "========\n")
```

Comment:
Should this function be moved to IS?  As of now, no one would be able to implement this method there without causing a method overwrite.

Reply:
I don't see why we would send this to IS. It seems unnecessary and it is something it should be easy to adjust as the wording changes